docs-ref-services/latest/storage.md

```diff
@@ -349,7 +349,25 @@ The `BlobClient.upload(InputStream data)` method overwrites an existing blob by
 
 | Client library | Versions impacted | Minimum safe version | Recommended action |
 | --- | --- | --- | --- |
-| Azure Storage Blob | 12.20.0-beta.1 to 12.29.0-beta.1 | 12.29.0-beta.2 | [Update to latest version or minimum 12.29.0-beta.2](https://mvnrepository.com/artifact/com.azure/azure-storage-blob) |
+| Azure Storage Blob | 12.20.0-beta.1 to 12.29.0-beta.1 | 12.29.0 | [Update to latest version or minimum 12.29.0](https://mvnrepository.com/artifact/com.azure/azure-storage-blob) |
+
+#### Recommended steps
+
+Update client library versions according to the table above.
+
+[Back to list of known issues](#list-of-known-issues)
+
+### 12. Downloading with `ShareFileClient.downloadToFile()` can write incorrect data to a file
+
+#### Issue description
+
+In the **azure-storage-file-share** package, the `ShareFileClient.downloadToFile()` method can write incorrect data to a file when more than 5 retries occur due to partial network responses while streaming data. The maximum number of retries is unintentionally higher (up to a maximum of 15), which can result in writing to incorrect positions in the file when the operation is retried more than 5 times. The data in Storage is still correct.
```

Comment:
`The data in Storage is still correct.`

Maybe we should say "This issue does not affect data in the Storage service, only the data being written to the local filesystem."

Reply:
Updated. thx